client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -37,6 +37,10 @@ class SelectBusinessType extends Component {
 		);
 	};
 
+	trackGoogleMyBusinessLinkClick = () => {
+		this.props.recordTracksEvent( 'calypso_google_my_business_select_business_type_link_click' );
```

Comment:
I know it's beginning to be quite long, but if we want to stick to the naming convention the name of this event should probably be:

```
calypso_google_my_business_select_business_type_google_my_business_link_click
```